src/connectedmachine/azext_connectedmachine/generated/_help.py

```diff
@@ -19,7 +19,7 @@
 
 helps['connectedmachine list'] = """
     type: command
-    short-summary: "List all the hybrid machines in the specified resource group. Use the nextLink property in the \
+    short-summary: "Lists all the hybrid machines in the specified resource group. Use the nextLink property in the \
```

Comment:
We should use the first person in the help description, so please do not modify these voices. 
I see many similar changes below, please don't make such changes~